fimsauthz-api/fims-authz.yaml

```diff
@@ -283,14 +283,64 @@ components:
       type: object
       additionalProperties: false
       properties:
-        jose_header:
-          $ref: "#/components/schemas/JoseHeader"
-        jws_payload:
+        header:
+          $ref: "#/components/schemas/xUtmMessageSignatureJoseHeader"
+        payload:
+          type: object
+          description: >-
+            The payload of the JWS will be the same character string as the HTTP body.
+        signature:
+          type: string
+          description: >-
+            The signature is calculated on the payload based on the information in the header.
+    xUtmMessageSignatureJoseHeader:
+      type: object
```

Comment:
Add description which says what this element is relative to the Http Response data structure